scripts/release.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+atlas_api="https://atlas.hashicorp.com/api/v1/box/hashbang/shell-server"
+atlas_token=$(jq -r '.atlas_token' config.json)
+bintray_user=$(jq -r '.bintray_user' config.json)
+bintray_key=$(jq -r '.bintray_key' config.json)
+version="$VERSION"
+
+if [ ! -z "$VERSION+x" ]; then
+	printf "Version number: "
+	read version
+fi
+
+for file in dist/*; do \
+	test -h $file && continue;
+	artifact_file=$(basename $(readlink -f $file))
+	artifact_name=${artifact_file%%.*}
+	artifact_type=${artifact_name%-*}
+	artifact_timestamp=${artifact_name##*-}
+
+	gpg \
+		--local-user D2C4C74D8FAA96F5 \
+		--detach-sig \
+		$file
+
+	if [ "$artifact_type" = "docker" ]; then
+		docker_user=$(jq -r '.docker_user' config.json)
+		docker_pass=$(jq -r '.docker_pass' config.json)
+		docker login -u $docker_user -p $docker_pass
+		docker import - "hashbang/shell-server:$artifact_version" < "$file"
```

Comment:
starting to sound like a broken record: should `hashbang/shell-server` be user configurable?